google/resource_compute_instance_group_manager.go

```diff
@@ -132,18 +132,18 @@ func resourceComputeInstanceGroupManagerCreate(d *schema.ResourceData, meta inte
 		return err
 	}
 
-	// Get group size, default to 1 if not given
-	var target_size int64 = 1
+	var targetSize int64 = 0
```

Comment:
Go is convinced this is an `int`, not `int64` with that syntax; using `targetSize := int64(0)`